spec.md

```diff
@@ -246,8 +249,9 @@ Policies MUST specify the following fields:
   clients SHOULD cache a policy for up to this value from last policy fetch
   time.
 * rua: Address to which aggregate feedback MAY be sent (comma-separated
-  plain-text list of email addresses, optional). For example,
-  "mailto:postmaster@example.com" from [@!RFC3986].
+  plain-text list of email addresses or an HTTPS endpoint, optional). For example,
+  "mailto:postmaster@example.com" from [@!RFC3986] or HTTP POST to "https://example.com/sts-report".
+* id: A unique identifier to identify a policy. It could be first 16 bytes of SHA256(all fields except 'id') hash
```

Comment:
One comment about this: Since we already specified that reports contain the raw string of the applied policy, is there utility in having a policy *id*? It seems to achieve the same thing, but with a layer of indirection (which may be good or bad; I'm not sure).